code/game/machinery/kitchen/gibber.dm

```diff
@@ -208,7 +208,8 @@
 	slab_nutrition /= slab_count
 
 	for(var/i=1 to slab_count)
-		var/obj/item/new_meat = new slab_type(src, rand(3,8))
+		var/obj/item/drop = new slab_type(src, rand(3,8))
+		var/obj/item/weapon/reagent_containers/food/snacks/meat/new_meat = drop

```

Comment:
Why is drop needed at all?
Won't this work?
```dm
var/obj/item/weapon/reagent_containers/food/snacks/meat/new_meat = new slab_type(src, rand(3,8))
```

Reply:
It was runtiming due to ipc's meat.

Reply:
What runtime?

Reply:
Your suggestion above wont work, because the IPC meat is defined as `/obj/item/stack/material/steel`
(As noted above)

Reply:
My suggestion is functionally identical to what's here if `drop` isn't used anywhere else.

Reply:
I do not believe that
`var/obj/item/weapon/reagent_containers/food/snacks/meat/new_meat = new slab_type(src, rand(3,8))` is going to work, if slab_type is a `/obj/item/stack/material/steel`

Reply:
I tested and lohikar's suggestion seems to work.